circuit/types/constants.go

```diff
@@ -39,7 +39,12 @@ const (
 	DefaultInt = int64(-1)
 
 	NbAccountAssetsPerAccount = 4
-	NbAccountsPerTx           = 5
+	NbAccountsPerTx           = 4
+
+	GasAccountIndex = 1 // gas account index
```

Comment:
GasAccountIndex should be a variable rather than const

Reply:
sure